RCovidSummary.R

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env Rscript
+# Author: JLinares/2022
+# Usage: 1. analysis summary for PatientSamples: Two files are generated: Summary and RunParameters
+#        2. Arguments: run name
+#        3. Example: Rscript / . . . ./summary_v2.R UT-A01290-220416
+#        4. Connditions/precautions:
+#           a: Analysis completed and Cecret/Lineage_report.csv completed before running this script
+#           b: The samplesIds, generated using a Randon Number Generator, will be differents everytime this sctript is ran.
```

Comment:
I would also add where the output is going to be. I know its at the bottom but I missed it when I ran the script and thought that it would be in the directory I ran the script in.